woocommerce/woocommerce-ios
# SPE: Add Simplified Product Editing experiment

Closes: #8999 


## Description


This adds the A/B experiment for Simplified Product Editing (SPE) and enables the SPE behavior for the experiment treatment group. Experiment details (internal ref): pbxNRc-2li-p2

The SPE feature flag is still in place, to ensure that the control behavior is used for UI testing. UI tests will be updated and the feature flag removed if the treatment behavior is successful in the A/B experiment.

FYI @wzieba 

## Testing instructions


### Prerequisite (before launching the app)

1. Open the experiment details in Abacus (linked at the top of pbxNRc-2li-p2 or ping me for the direct link).
2. In the Audience section of the experiment setup, go to Variations > Manual Assignment and use the bookmarklet to assign yourself to the treatment group.

### Test the treatment behavior

1. Build and run the app. **Note:** You must be logged in to the app with your a8c account to test the experiment assignment/behavior while the experiment is in staging. If you are logged in to a different account, log out and log in again with your a8c account.
2. Go to the Products tab and add a new product.
3. Confirm you see the expected treatment behavior (new SPE experience).

### Test the control behavior

1. Log out and log in with a non-a8c test account. (Alternately, use the prerequisite steps to assign yourself to the control group to use your a8c account. While the test is in staging, manually assigning yourself to the control group or logging in with a non-a8c account will show you the control behavior.)
2. Go to the Products tab and add a new product.
3. Confirm you see the expected control behavior (same as production).

## Screenshots


### Treatment

https://user-images.githubusercontent.com/8658164/221851502-f1650e69-3af8-4fdc-a1db-b9da796ecd22.mp4


### Control


https://user-images.githubusercontent.com/8658164/221851643-eca03f34-25bb-47a2-a64b-e6d4a5079f0c.mp4



---
